newsletter-december-2023.rst

```diff
@@ -0,0 +1,41 @@
+.. post:: Dec 6, 2022
+   :tags: newsletter, python
+   :category: Newsletter
+   :author: Eric
+
+Read the Docs newsletter - December 2023
+========================================
+
+News and updates
+----------------
+
+* We have shipped :doc:`single version projects </introducing-support-for-version-only-projects>` to allow projects to be versioned without having translations. This is a long-requested feature that we've excited to ship based on our Proxito refactor work.
+* We improved our webhook security by :doc:`requiring a secret </security-update-on-incoming-webhooks>` to be configured for all webhooks. This will help prevent malicious actors from triggering builds and other actions.
+* Work continues on `Addons <https://github.com/readthedocs/addons/>`_, where we've added support for explicit EthicalAds placements on documentation pages, and are working on a new interface for configuring Addons.
+
```

Comment:
We have two security advisories

- https://github.com/readthedocs/readthedocs.org/security/advisories/GHSA-9vh9-cxm2-p2c4
- https://github.com/readthedocs/readthedocs.org/security/advisories/GHSA-45hq-g76r-46wv

Reply:
Added 👍